template.go

```diff
@@ -22,16 +23,15 @@ var exeArg [][]string = [][]string {
 {{- end}}{{end -}}
 }
 
-func printClose(r io.ReadCloser, prefix string){
+func drainPipe(r io.ReadCloser, prefix string){
 	defer r.Close()
-	slurp, _ := io.ReadAll(r)
-	if len(slurp) == 0 {
-		return
+
+	scanner := bufio.NewScanner(r)
+	for scanner.Scan() {
+		fmt.Printf("[            ] %s: %s\n", prefix, scanner.Text())
 	}
-	fmt.Printf("[            ] %s\n%s", prefix, slurp)
 }
```

Comment:
A call to [(*Scanner) Err()](https://pkg.go.dev/bufio#Scanner.Err) is missing after the loop exit from [(*Scanner) Scan()](https://pkg.go.dev/bufio#Scanner.Scan). Otherwise every error that might happen will be ignored.

Reply:
👍 ty for the catch, done